zipline/utils/paths.py

```diff
@@ -0,0 +1,223 @@
+"""
```

Comment:
I made the defaults for os.environ more standard. They used to be None and then bind to os.environ.copy().